### PR TITLE
Generate ipk package file for toltec

### PR DESCRIPTION
**Pull request by Eeems**

### CODE REVIEW

File: package [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        -p:Configuration=Release \
+        -p:IncludeAllContentForSelfExtract=true \
+        -p:PublishReadyToRunShowWarnings=true \
+        src/ReTerm/ReTerm.csproj
+}
+
+package() {
+    install -D -m 755 -t "$pkgdir"/opt/bin "$srcdir"/src/ReTerm/bin/Release/net6.0/linux-arm/publish/ReTerm
+    install -D -m 644 -t "$pkgdir"/opt/etc/draft "$srcdir"/draft/reterm.draft
+    install -D -m 644 -t "$pkgdir"/opt/etc/draft "$srcdir"/draft/reterm.png
```
Comment (i-am-shodan) [inline package]: Do you also need to make ReTerm.sh executable?

Comment (i-am-shodan) [reply package]: @Eeems bump

Comment (Eeems) [reply package]: I guess I'd have to also include ReTerm.sh

**Comment by i-am-shodan** [comment]
I'll approve this when I've fixed a rendering bug. Probably tomorrow

**Comment by Eeems** [comment]
I'm tweaking the build/deployment location at the moment as well. Should be done in a bit.

**Comment by Eeems** [comment]
The TerminalFont change is untested, but in theory should work. I can't properly test this as I don't have an external keyboard setup for my rM1.

**Pull request merged**